Training/Easy/power_of_thor.rb

```diff
@@ -7,58 +7,52 @@ def initialize
     @light_x, @light_y, @x, @y = gets.split(' ').collect(&:to_i)
```

Comment:
Don't call `gets` here. Instead, call it from outside the object and give it to an argument to the `initialize` method